packages/dynamite/dynamite_end_to_end_test/pubspec.yaml

```
@@ -0,0 +1,34 @@
+name: dynamite_end_to_end_test
+publish_to: none
+description: Tests for dynamite. Not meant for publishing.
+version: 1.0.0
+
+environment:
+  sdk: '>=3.1.1 <4.0.0'
+
+dependencies:
+  built_collection: ^5.1.1
+  built_value: ^8.6.2
+  collection: ^1.18.0
+  dynamite_runtime:
+    git:
+      url: https://github.com/nextcloud/neon
+      path: packages/dynamite/dynamite_runtime
+  meta: ^1.10.0
+  universal_io: ^2.2.2
+
+dev_dependencies:
+  build_runner: ^2.4.6
+  build_test: ^2.2.1
+  build_verify: ^3.1.0
+  built_value_generator: ^8.6.2
+  dynamite:
+    git:
+      url: https://github.com/nextcloud/neon
+      path: packages/dynamite/dynamite
+  lints: ^2.0.0
+  neon_lints:
+    git:
+      url: https://github.com/nextcloud/neon
+      path: packages/neon_lints
+  test: ^1.21.0
```

```suggestion
  test: ^1.24.8
```
No reason to not use the latest version